docs/reference/extensions/django-framework.rst

```diff
@@ -45,13 +45,14 @@ application. In the following example we use it to specify ``libpq-dev``:
 
 .. warning::
   You can only use 1 of the dependencies parts at a time.
+  (eg. `parts-django-framework-async-dependencies`_ or
+  `parts-django-framework-dependencies-stage-packages`_)
```

Comment:
I'm not sure putting it like this makes sense, it could be implying that the stage packages or limited to 1. Perhaps review this documentation with @erinecon?